src/test/app/PayStrand_test.cpp

```diff
@@ -857,6 +844,30 @@ struct PayStrand_test : public beast::unit_test::suite
             }
         }
     }
+    void
+    run() override
+    {
+        testAllPairs({featureFlow, fix1373});
+        testAllPairs({featureFlow, fix1373, featureFlowCross});
+    }
+};
+
+BEAST_DEFINE_TESTSUITE_MANUAL(PayStrandAllPairs, app, ripple);
+
+struct PayStrand_test : public beast::unit_test::suite
+{
+    static bool hasFeature(uint256 const& feat)
```

Comment:
I think this method is no longer needed since you are now using `std::initializer_list` to pass `features` rather than variadic templates.